apps/gitness/src/pages/repo-sandbox/repo-sandbox-list.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+  Button,
+  ListActions,
+  ListPagination,
+  Pagination,
+  PaginationContent,
+  PaginationItem,
+  PaginationLink,
+  PaginationNext,
+  PaginationPrevious,
+  SearchBox,
+  Spacer,
+  Text
+} from '@harnessio/canary'
+import { useState } from 'react'
+import { useListReposQuery, RepoRepositoryOutput } from '@harnessio/code-service-client'
+import { SkeletonList, RepoList, SandboxLayout } from '@harnessio/playground'
+import { Link, useNavigate } from 'react-router-dom'
+import { useGetSpaceURLParam } from '../../framework/hooks/useGetSpaceParam'
+import { usePagination } from '../../framework/hooks/usePagination'
+import { timeAgoFromEpochTime } from '../pipeline-edit/utils/time-utils'
+
+const filterOptions = [{ name: 'Filter option 1' }, { name: 'Filter option 2' }, { name: 'Filter option 3' }]
+const sortOptions = [{ name: 'Sort option 1' }, { name: 'Sort option 2' }, { name: 'Sort option 3' }]
+const viewOptions = [{ name: 'View option 1' }, { name: 'View option 2' }]
+
+const LinkComponent = ({ to, children }: { to: string; children: React.ReactNode }) => <Link to={to}>{children}</Link>
+
+export default function ReposSandboxListPage() {
+  // hardcoded
+  const totalPages = 10
+  const navigate = useNavigate()
+  const space = useGetSpaceURLParam()
+  const [loadState, _] = useState('float')
+
+  const { isFetching, data } = useListReposQuery({ queryParams: {}, space_ref: `${space}/+` })
+  const { currentPage, previousPage, nextPage, handleClick } = usePagination(1, totalPages)
+
+  const renderListContent = () => {
+    if (isFetching) {
+      return <SkeletonList />
+    }
+    return (
+      data && (
+        <RepoList
+          LinkComponent={LinkComponent}
+          repos={data?.map((repo: RepoRepositoryOutput) => {
+            return {
+              id: repo.id,
+              name: repo.identifier,
+              description: repo.description,
+              private: !repo.is_public,
+              stars: 0,
+              forks: repo.num_forks,
+              pulls: repo.num_pulls,
+              timestamp: repo.updated && timeAgoFromEpochTime(repo.updated)
+            }
+          })}
+        />
+      )
+    )
+  }
+
+  return (
+    <>
+      {loadState.includes('sub') && (
```

Comment:
This conditional subpanel is from Playground's demo states (the bottom right ellipsis) - probably you don't want that in a product, though?